### ISSUE TITLE
AboutModal image alt text

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1058]
For the image that displays on the right side of the About Modal, the following alt text is defined:
"Product Logo"

Instead, we should update this to represent best practice (i.e. describe the image as a "logo", but also repeat any text included in the image). The following text should be used instead: 
"Patternfly Logo"

**Comment by jgiardino** [comment]
@stacymcauliffe @mcarrano 
There's an inconsistency between pf core and pf-react with the first image that displays in the about modal. In core, it's "Patternfly brand logo" and in pf-react it's "Patternfly Logo". Do you have a preference for one of these? I'd like to have them be consistent just to avoid questions in the future.

![image](https://user-images.githubusercontent.com/21063328/49409281-e9a08400-f72d-11e8-87b8-559c56bbe549.png)

BTW, I have issues for both core (this one) and react (https://github.com/patternfly/patternfly-react/issues/1008) to change the alt text for the following image to "Patternfly Logo". Please comment if you suggest otherwise.

![image](https://user-images.githubusercontent.com/21063328/49409334-308e7980-f72e-11e8-925a-c2c458e1c5fb.png)

**Comment by stacymcauliffe** [comment]
@jgiardino - I think PatternFly logo is fine. Do we want to standardize on sentence style caps for alt text?

**Comment by mcarrano** [comment]
I think this gets into the question of what these two logo spaces are for @jgiardino . I believe that what's called "Patternfly brand logo" is intended to be there to brand the product as made with PatternFly while the "PatternFly logo" graphic is just a placeholder that a consumer would replace with their own graphic. Can you confirm that @kybaker ? This might influence what should come first. Agree that a consistent approach would be good.

**Comment by matthewcarleton** [comment]
closing this now b/c the about modal no longer has this image on the right side.
